Utils/ExtractVersionFromTilsesetJSON/compute_versiondata_from_py3dfiles.py

```diff
@@ -30,12 +31,15 @@ def log(msg):
         print(msg)
 
 
-# load data
-# /!\ The list of transactions are expected to be found in extensions-3DTILES_temporal-transactions
-def load_tilesetJSON():
+"""
+# Load the file .\data\tileset.json as a json
```

Comment:
Please update this comment since it loads the tileset passed as an argument now.